src/OpenSearch.Client/ApiUrlsLookup.cs

```diff
@@ -172,9 +172,6 @@ internal static partial class ApiUrlsLookups
 		internal static ApiUrls SnapshotRestore = new ApiUrls(new[]{"_snapshot/{repository}/{snapshot}/_restore"});
 		internal static ApiUrls SnapshotStatus = new ApiUrls(new[]{"_snapshot/_status", "_snapshot/{repository}/_status", "_snapshot/{repository}/{snapshot}/_status"});
 		internal static ApiUrls SnapshotVerifyRepository = new ApiUrls(new[]{"_snapshot/{repository}/_verify"});
-		internal static ApiUrls TasksCancel = new ApiUrls(new[]{"_tasks/_cancel", "_tasks/{task_id}/_cancel"});
```

Comment:
why? Is it removed on OpenSearch side?

Reply:
It's moved to here: [src/OpenSearch.Client/_Generated/ApiUrlsLookup.cs](https://github.com/opensearch-project/opensearch-net/pull/320/files#diff-b857fe2c5d34de5b280327bcea9d1272e7c5976334d34844d809971bfaac5ada)